<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmbScanner</class>
 <widget class="QWidget" name="SmbScanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="discoverTitleLabel">
     <property name="text">
      <string>SMB Discover</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="discoverFrame">
     <property name="styleSheet">
      <string notr="true">#discoverFrame {
	border: 1px solid #aaa;
	border-radius: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="discoverDescLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Scans the target network looking for hosts with SMB active using the &lt;span style=&quot; font-weight:600;&quot;&gt;nbtscan&lt;/span&gt; scanner.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="controlsLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="repeatsLabel">
          <property name="text">
           <string>Restransmits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="repeatsSpin"/>
        </item>
        <item>
         <widget class="QCheckBox" name="collectSharesCheckbox">
          <property name="text">
           <string>Collect sha&amp;res on finish</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="discoverButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Discover hosts with S&amp;MB active</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="discoverOutputTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#discoverOutputTextEdit {
	background-color: rgb(83, 83, 83);
	color: rgb(226, 226, 226);
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Discover commands output</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="sharesCollectorLabel">
     <property name="text">
      <string>Shares Collector</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="collectorFrame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#collectorFrame {
	border: 1px solid #aaa;
	border-radius: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="collectorDescLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Collect open shares for scanned hosts with &lt;span style=&quot; font-weight:600;&quot;&gt;smbclient &lt;/span&gt;and &lt;span style=&quot; font-weight:600;&quot;&gt;smblibrary &lt;/span&gt;programs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QProgressBar" name="collectorProgressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="collectSharesButton">
          <property name="text">
           <string>Col&amp;lect SMB shares</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="collectorOutputTextEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#collectorOutputTextEdit {
	background-color: rgb(83, 83, 83);
	color: rgb(226, 226, 226);
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Collector commands output</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
